WordPress/WordPress.xcodeproj/xcshareddata/xcschemes/Jetpack.xcscheme

```diff
@@ -121,11 +121,6 @@
             value = ""
             isEnabled = "YES">
          </AdditionalOption>
-         <AdditionalOption
-            key = "MallocScribble"
-            value = ""
-            isEnabled = "YES">
-         </AdditionalOption>
```

Comment:
Just want to highlight this scheme change, which I think should be done: disabling Malloc Scribble diagnosis prevents WKWebView from crashing during debug sessions.